Copy to SD Card root directory to update/config.ini

```diff
@@ -37,6 +37,7 @@
 #   M115_GEOMETRY_REPORT (in Configuration_adv.h)
 #   M114_DETAIL (in Configuration_adv.h)
 #   REPORT_FAN_CHANGE (in Configuration_adv.h)
+#   AUTO_REPORT_POSITION (in Configuration_adv.h)
```

Comment:
the parameters were listed in the same order they appear in Configuration.h / Configuration_adv.h. So AUTO_REPORT_POSITION should be provided just before M115_GEOMETRY_REPORT